### PR TITLE
Support minimum price for Order (closes #1447)

### LINKED ISSUE #1447: Ability to specify minimum price per for trading

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

File: app/models/market.rb [hunk 1]
```diff
@@ -133,6 +135,8 @@ def units_must_be_enabled
 #  enabled       :boolean          default(TRUE), not null
 #  created_at    :datetime         not null
 #  updated_at    :datetime         not null
+#  min_bid_price :decimal(17, 16)  default(0.0), not null
+#  min_ask_price :decimal(17, 16)  default(0.0), not null
```
Comment (ysv) [inline app/models/market.rb]: place them after `bid_fee`

File: spec/models/order_ask_spec.rb [hunk 2]
```diff
@@ -36,5 +38,12 @@
         OrderBid.new(volume: '31'.to_d, ord_type: 'market').compute_locked
       end.to raise_error(RuntimeError, 'Market is not deep enough')
     end
+
+    it 'should make sure price is greater than min_ask_price' do
+      market.update(min_ask_price: 0.1)
```
Comment (ysv) [inline spec/models/order_ask_spec.rb]: Move this to `let(:market)`
```ruby 
let(:market) do
 Market.find(:btcusd).tap { |m| m.update(...)}
end
```

**Review by ysv — CHANGES_REQUESTED** [review]

File: spec/models/order_bid_spec.rb [hunk 3]
```diff
@@ -45,5 +47,12 @@
         OrderBid.new(volume: '31'.to_d, ord_type: 'market').compute_locked
       end.to raise_error(RuntimeError, 'Market is not deep enough')
     end
+
+    it 'should make sure price is greater than min_bid_price' do
+      market.update(min_bid_price: 0.1)
```
Comment (ysv) [inline spec/models/order_bid_spec.rb]: Same as [here](https://github.com/rubykube/peatio/pull/1460/files#r204420540)

File: app/models/market.rb (hunk 1 above)
Comment (ysv) [reply app/models/market.rb]: also don't forget to annotate models

**Comment by mod** [comment]
I think it should be:
min_ask
max_bid

**Comment by mod** [comment]
@shal please update

**Comment by shal** [comment]
@mod Updated and rebased.

**Comment by shal** [comment]
@mod it is ready.

**Pull request merged**